modules/formats/from-env.nu

```diff
@@ -0,0 +1,12 @@
+# Converts a .env file into a record
+# may be used like this: open .env | load-env
+# works with quoted and unquoted .env files
+def "from env" []: string -> record {
+  lines 
+    | split column '#' # remove comments
+    | get column1 
+    | filter {($in | str length) > 0} 
```

Comment:
I have no problem landing this but I think this line is removing blank lines. You could do the same with the `compact` command.

Reply:
I realized that the `parse` command already gets rid of the empty lines, so I removed that.